frontend/src/metabase/reference/reference.js

```diff
@@ -67,6 +67,19 @@ const HIDE_DASHBOARD_MODAL = "metabase/reference/HIDE_DASHBOARD_MODAL";
 export const hideDashboardModal = createAction(HIDE_DASHBOARD_MODAL);
 
 
+// X-Ray whatnotery
+const FETCH_FIELD_FINGERPRINT = 'metabase/reference/FETCH_FIELD_FINGERPRINT';
+const fetchFieldFingerPrint = createThunkAction(FETCH_FIELD_FINGERPRINT, function(fieldId) {
+    return async () => {
+        try {
+            let fingerprint = await MetabaseAPI.fieldFingerPrint(fieldId)
```

Comment:
In general the action/reducer logic looks good, just some small fixes needed:
- MetabaseAPI isn't imported; add `import { MetabaseApi } from "metabase/services"` in the beginning of file. 
- Name the method to `field_fingerprint` in services.js for consistency with other API method names.